pramen/core/src/test/resources/log4j.properties

```diff
@@ -32,3 +32,4 @@ log4j.logger.za.co.absa.pramen.core.state.PipelineStateImpl=OFF
 log4j.logger.za.co.absa.pramen.core.utils.ConfigUtils$=OFF
 log4j.logger.za.co.absa.pramen.core.utils.JdbcNativeUtils$=OFF
 log4j.logger.za.co.absa.pramen.core.utils.SparkUtils$=OFF
+log4j.logger.za.co.absa.pramen.core.utils.ThreadUtils$=OFF
```

Comment:
Could you please explain why you did it?

Reply:
This is so running test suites won't pollute the screen with errors that are expected